nba/views.py

```diff
@@ -16,7 +18,17 @@ def players(request):
 # A view for each player
 def player(request, player_id):
     player = Player.objects.get(playerid=player_id)
-    return HttpResponse("{}".format(player))
+    games_list = PlayerGameStat.objects.all().filter(playerid = player_id).order_by('-gameid__date')
```

Comment:
Just a nit pick here no changes required, this is just for future reference

This same query can actually be done easier by using reverse accessors on django models
```python
games_list = player.playergamestats.order_by('-gameid__date')
```
basically with a "has many" relationship, django knows that Player has many games associated with it, so each player object can have it's game stats accessed as if they were an attribute of the Player object

Reply:
That's sweet. Way simpler